### DISCUSSION TITLE
Add reference of blogpost to pdf

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#5569] [Q&A] [answered]
I am using multiple formats for my blog (html and pdf)

format: 
  html: default
  pdf: default

I would like the pdf to inherit the citation that I have in my html output (I am using citation: true).

**Comment by mcanouil** [comment]
I don't know what you are calling "citation".
Shared/global option are either top-level YAML in your document frontmatter or place top-level in `_quarto.yml`.

---

If you want more help, please consider sharing a small "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by mcanouil** [reply]
`citation` is for the document itself.
Thus for your PDF, it can't be the reference for your HTML.

The only to add that reference, would be to create it as YAML or bib latex entry and add that to your PDF.

**Comment by ghost** [comment]
> The only to add that reference, would be to create it as YAML or bib latex entry and add that to your PDF.

Can you put an example of this? So in the case of bilatex you mean that I should cite my post somewhere in my document to make it appear in the references section in the pdf output? 

In the part of the YAML I do not understand what I should put there.

**Reply by mcanouil** [reply, marked as answer]
Get the bib latex code:

```md
@book{knuth1986texbook,
  author    = {Donald E. Knuth},
  title     = {The {\TeX}book},
  year      = {1986},
  publisher = {Addison-Wesley},
  address   = {Reading, MA},
  edition   = {1st},
  isbn      = {0-201-13447-0},
}
```

And use it in your PDF the way you want.

See:
- <https://quarto.org/docs/authoring/create-citeable-articles.html>
- <https://quarto.org/docs/authoring/footnotes-and-citations.html>
